testnets/holesky/README.md

```diff
@@ -559,13 +594,14 @@ sidecar. You can see them in your terminal by running the Bolt sidecar binary
 with the `--help` flag:
 
 ```
+
 Command-line options for the Bolt sidecar
 
 Usage: bolt-sidecar [OPTIONS] --validator-indexes <VALIDATOR_INDEXES> --engine-jwt-hex <ENGINE_JWT_HEX> --fee-recipient <FEE_RECIPIENT> --builder-private-key <BUILDER_PRIVATE_KEY> --commitment-private-key <COMMITMENT_PRIVATE_KEY> <--constraint-private-key <CONSTRAINT_PRIVATE_KEY>|--commit-boost-signer-url <COMMIT_BOOST_SIGNER_URL>|--keystore-password <KEYSTORE_PASSWORD>|--keystore-secrets-path <KEYSTORE_SECRETS_PATH>>
 
 Options:
-      --port <PORT>
-          Port to listen on for incoming JSON-RPC requests of the Commitments API. This port should be open on your firewall in order to receive external requests!
+--port <PORT>
+Port to listen on for incoming JSON-RPC requests of the Commitments API. This port should be open on your firewall in order to receive external requests!
```

Comment:
I think this formatting is wrong, can you re-add this spacing? also below

Reply:
Thanks for noticing!